features/collection/thresholding/impl/src/main/java/org/opennms/netmgt/threshd/ExpressionConfigWrapper.java

```diff
@@ -166,7 +166,10 @@ public double evaluate(String expression, Map<String, Double> values) throws Thr
             // Fetch an instance of the JEXL script engine to evaluate the script expression
             Object resultObject = jexlEngine.createExpression(expression).evaluate(new MapContext(context));
             result = Double.parseDouble(resultObject.toString());
-        } catch (Throwable e) {
+        } catch (ArithmeticException e) {
+            LOG.error("ArithmeticException while evaluating expression " + m_expression.getExpression() + ": " + e.getMessage());
```

Comment:
Hi @dino2gnt, in order to reduce the number of these error messages in log, we want to use the log rate limiter; I think it might make sense to keep it as an "error", because after all an exception is thrown, that will not be handled in the Catch clause. On the other side, if this Exception is thrown because of some user entering a wrong value in the frontend interface, than we might consider it as "warning"